hive/src/main/java/io/confluent/connect/storage/hive/HiveSchemaConverter.java

```diff
@@ -26,11 +29,16 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.apache.kafka.connect.data.Time;
+import org.apache.kafka.connect.data.Timestamp;
 
 public class HiveSchemaConverter {
 
   private static final Map<Type, TypeInfo> TYPE_TO_TYPEINFO;
 
+  private static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
```

Comment:
Any new configuration options that need to be introduced here, will need to be defined along with the other ones for Hive in:
https://github.com/confluentinc/kafka-connect-storage-common/blob/master/hive/src/main/java/io/confluent/connect/storage/hive/HiveConfig.java

Reply:
This property is name is defined in `AvroData`. Here, I'm just using it to retrieve the value of precision, which was set when the runtime connect record was created using `AvroData`. I think users of Hive connector should be given the option to config this name.

Reply:
ok.